lib/fs/index.js

```diff
@@ -33,12 +35,13 @@ const api = [
   'unlink',
   'utimes',
   'writeFile'
-]
-// Add methods that are only in some Node.js versions
-// fs.copyFile was added in Node.js v8.5.0
-typeof fs.copyFile === 'function' && api.push('copyFile')
-// fs.mkdtemp() was added in Node.js v5.10.0
-typeof fs.mkdtemp === 'function' && api.push('mkdtemp')
+].filter(key => {
+  // Some commands are not available on some systems. Ex:
+  // fs.copyFile was added in Node.js v8.5.0
+  // fs.mkdtemp was added in Node.js v5.10.0
+  // fs.lchown is not available on Linux
+  return fs[key]
```

Comment:
Just for consistency with the previous code, to prevent possible regressions, can we change this to 

    return typeof fs[key] === 'function'

?